Scenarios/Scenario.swift

```diff
@@ -17,13 +17,22 @@ public final class Scenario: Preparable, Actionable, ScenarioBuilder {
   private let commitFunc: CommitFunc
   private var stepDescriptions: [StepMetadata] = []
 
-  public init(_ name: String, file: String = #file, line: UInt = #line, commit: @escaping CommitFunc = quick_it) {
+  public init(_ name: String, file: String = #file, line: UInt = #line, commit: @escaping CommitFunc) {
     self.name = name
     self.file = file
     self.line = line
     self.commitFunc = commit
   }
 
+  public convenience init(_ name: String, file: String = #file, line: UInt = #line) {
```

Comment:
I like the looks of this separation. What was the motivation behind preferring the use of a convenience initialiser, over a default parameter? May I hazard a guess and say it's for autocompletion and visibility?